pkg/subctl/cmd/join.go

```diff
@@ -1,13 +1,44 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
+
+	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/deploy"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/install"
 )
 
 func init() {
 	rootCmd.AddCommand(joinCmd)
 }
 
+const (
+	SubmarinerNamespace = "submariner"
+)
+
 var joinCmd = &cobra.Command{
-	Use: "join",
+	Use:   "join",
+	Short: "connect a cluster to an existing broker",
+	Args:  cobra.ExactArgs(1), // exactly one, the broker data file
+	Run: func(cmd *cobra.Command, args []string) {
+
```

Comment:
Nit: Remove newline at start of function to be consistent with other relevant code.